lib/twilio/errors/generated.ts

```diff
@@ -252,6 +252,85 @@ export namespace GeneralErrors {
   }
 }
 
+export namespace MalformedRequestErrors {
+  export class MalformedRequestError extends TwilioError {
+    causes: string[] = [
+      'No CallSid in the message object.',
+      'No VoiceEventSid in the message object.',
+      'No payload in the message object.',
+      'Invalid or missing payload in the message object.',
+      'No message type in the message object.',
+    ];
+    code: number = 31100;
+    description: string = 'The request had malformed syntax.';
+    explanation: string = 'The request could not be understood due to malformed syntax.';
+    name: string = 'MalformedRequestError';
+    solutions: string[] = [
+      'Ensure the message object contains a valid CallSid.',
+      'Ensure the message object contains a valid VoiceEventSid.',
+      'Ensure the message object has a valid payload.',
+      'Ensure the message object has a valid message type.',
+    ];
+
+    constructor();
+    constructor(message: string);
+    constructor(error: Error | object);
+    constructor(message: string, error: Error | object);
+    constructor(messageOrError?: string | Error | object, error?: Error | object) {
+      super(messageOrError, error);
+      Object.setPrototypeOf(this, MalformedRequestErrors.MalformedRequestError.prototype);
+
+      const message: string = typeof messageOrError === 'string'
+        ? messageOrError
+        : this.explanation;
+
+      const originalError: Error | object | undefined = typeof messageOrError === 'object'
+        ? messageOrError
+        : error;
+
+      this.message = `${this.name} (${this.code}): ${message}`;
+      this.originalError = originalError;
+    }
+  }
+}
+
+export namespace AuthorizationErrors {
```

Comment:
Why is this namespace getting exported twice?

Reply:
That's a good question. I believe it's because the top-most error codes `3xxxx` and `2xxxx` are different. We would need to modify the error generation script to fix this. Will two exported namespaces with the same label work? Let me try in browser to see what happens.

Reply:
It doesn't affect our ability to instantiate the error class, it seems the duplicate namespaces are just merged.

Reply:
Great!